### PR TITLE
Add executing job support for `google_cloud_run_v2_job` resource

### PR DESCRIPTION
**Pull request by yanweiguo**
For https://github.com/hashicorp/terraform-provider-google/issues/13390.

Adding field `start_execution_token` and field `run_execution_token`  to resource `google_cloud_run_v2_job` to allow executing the job automatically once the job is created or updated. This is currently add to `terraform-provider-google-beta` provider only.

Also changed the docker images used in job examples to the right one.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
cloudrunv2: added fields `start_execution_token` and `run_execution_token` to resource `google_cloud_run_v2_job`
```

### CODE REVIEW

File: mmv1/products/cloudrunv2/Job.yaml [hunk 1]
```diff
@@ -226,6 +233,12 @@ properties:
         name: 'useDefault'
         description: |
           If True, indicates to use the default project's binary authorization policy. If False, binary authorization will be disabled.
+  - !ruby/object:Api::Type::String
+    name: start_execution_token
```
Comment (whaught) [inline mmv1/products/cloudrunv2/Job.yaml:237]: Do we want to include `run_execution_token` in the same PR? That might be the more common user scenario (they are oneof)

Comment (whaught) [reply mmv1/products/cloudrunv2/Job.yaml:237]: okay it look like that one isn't visible for v2 yet, we'll do that in the future.

**Review by whaught — APPROVED** [review]

**Review by paridhishah18 — APPROVED** [review]

**Review by whaught — APPROVED** [review]

**Review by NickElliot — CHANGES_REQUESTED** [review]
Two issues from the test: 
* the supplied `launch_stage` is being ignored, and returned as "GA" even though the POST request had "BETA"
* `start_execution_token` is not being returned by the API, I'm not certain if this is expected, but the field may need an `ignore_read: true` tag on it

File: mmv1/products/cloudrunv2/Job.yaml [hunk 2]
```diff
@@ -226,6 +233,22 @@ properties:
...
+    description: |-
+      A unique string used as a suffix creating a new execution upon job create or update. The Job will become ready when the execution is successfully started.
+      The sum of job name and token length must be fewer than 63 characters.
+    conflicts:
+      - run_execution_token
+    min_version: beta
+  - !ruby/object:Api::Type::String
+    name: run_execution_token
```
Comment (NickElliot) [inline mmv1/products/cloudrunv2/Job.yaml:245]:
```suggestion
    name: runExecutionToken
```

File: mmv1/products/cloudrunv2/Job.yaml [hunk 3]
```diff
@@ -226,6 +233,22 @@ properties:
         name: 'useDefault'
         description: |
           If True, indicates to use the default project's binary authorization policy. If False, binary authorization will be disabled.
+  - !ruby/object:Api::Type::String
+    name: start_execution_token
```
Comment (NickElliot) [inline mmv1/products/cloudrunv2/Job.yaml:237]:
```suggestion
    name: startExecutionToken
```

**Review by NickElliot — CHANGES_REQUESTED** [review]
need to have camelcase for fieldnames in the yaml :)

File: mmv1/products/cloudrunv2/Job.yaml (hunk 3 above)
Comment (yanweiguo) [reply mmv1/products/cloudrunv2/Job.yaml]: Oops.

**Review by whaught — APPROVED** [review]

**Review by NickElliot — APPROVED** [review]
LGTM!

**Pull request merged**